test/src/csg/TestCSGAxialSurfaceMeshGenerator.C

```diff
@@ -62,13 +62,22 @@ TestCSGAxialSurfaceMeshGenerator::generateCSG()
     std::unique_ptr<CSG::CSGSurface> surface_ptr =
         std::make_unique<CSG::CSGPlane>(surf_name, 0.0, 0.0, 1.0, coeffs[i]);
     auto & csg_plane = csg_obj->addSurface(surface_ptr);
+
     const auto region_direction = csg_plane.getHalfspaceFromPoint(centroid);
     auto halfspace =
         ((region_direction == CSG::CSGSurface::Halfspace::POSITIVE) ? +csg_plane : -csg_plane);
     cell_region &= halfspace;
+
+    // Next two lines are added for increasing test coverage, can be ignored
+    csg_obj->renameSurface(csg_plane, "surf");
+    csg_obj->renameSurface(csg_plane, surf_name);
   }
 
   csg_obj->updateCellRegion(csg_cell, cell_region);
 
+  // Next two lines are added for increasing test coverage, can be ignored
+  csg_obj->renameCell(csg_cell, "tri_cell");
+  csg_obj->renameCell(csg_cell, "square_cell");
+
```

Comment:
I feel like we should actually rename these though and make sure that the final name in the JSON output matches the rename.

They could be initialized with a fake name and then changed here to the correct name so that we don't have to update the expected output or another test that depends on these names